composer.json

```diff
@@ -28,6 +28,7 @@
         "illuminate/support": "^5.8 || ^6.0 || ^7.0 || ^8.0"
     },
     "require-dev": {
+        "phpunit/phpunit": "^7",
```

Comment:
phpunit is listed in `require` (not `reqiure-dev`) on composer.json of `orchestra/testbench` and its version depends on `orchestra/testbench`.

- https://github.com/orchestral/testbench/blob/4.x/composer.json

So should probably omit this.

Reply:
Hmm, this a commonly use for without phpunit install to global on local test.

An advantage local installs. 😀🤔

Reply:
Okay, i did omit phpunit.

Reply:
Okay, can you merge. 😸

Reply:
Yes I know, but this is irrelevant for global installations. Because composer always prefers the path `vendor/bin/*`

However, redundantly including indirectly dependent dependencies is not a bad practice if the versions are properly specified.

It seems that at least the version spec needs to be modified as Travis has failed on some tests.